source/docs/guides/EMV_3D_secure.md

```diff
@@ -561,7 +561,7 @@ trnApproved=1
 
 ## EMV 3D Secure API
 
-The EMV 3D Secure API provides a set of end points that you can utilize to perform 3DSecure authentication without immediately processing a payment.  This allows the card holder and credit card number to be authenticated and potentially stored or later referenced in a payment transaction.  Our Payments API supports accepting the 3DS Session Data a token to the card data utilized in a previously authenticated 3D Secure transaction.
+The EMV 3D Secure API provides a set of end points that you can utilize to perform 3DSecure authentication without immediately processing a payment.  This can be useful in instances where may want to authenticate a card holder before storing their credentials to a customer profile or for authenticating a transaction that may be processed at a later time.  Our Payments API supports accepting the 3DS Session Data token of a previously authenticated 3D Secure transaction as a reference to the card data and 3D Secure authentication results.
```

Comment:
where [you] may want
comma before 'or' after 'profile'
comma before 'and 3DS Secure' after 'card data'

Reply:
Updates